### DISCUSSION TITLE
Twitch: Button 'Open All Streams' (in browser) to collect channel points

### DISCUSSION THREAD

**Discussion by droserafire** [Chatterino/chatterino2#5657] [Ideas] [answered]
To be able to collect channel points automatically, you would need currently an extension like BetterTTV. It would be beneficial to collect channel points fast, if there is a button which would open all streams in a browser. 

In addition to all tabs:
Nice to have: Would work as well with option: 'show live tabs only' to just open the streams which are live at the moment.

**Comment by iProdigy** [comment, marked as answer]
Twitch only lets you claim watch time channel points in up to 2 channels at a time, so opening every single channel would be overly heavy handed to achieve your use case

**Reply by droserafire** [reply]
Thank for this information. 
Does that mean that applies also to lurk that only 2 streamer would benefit from lurking?

**Reply by iProdigy** [reply]
Benefit the streamer in what sense? Lurking in chatterino doesn't impact the streamer but leaving multiple streams running can raise more than two channels' viewcounts
